autosilk-cli.yaml

```diff
@@ -0,0 +1,12 @@
+project-name: cfparams-reporting
+pipeline-slug: cfparams
+cluster-name: cfparams
+github-repo: cfparams
+commit-branch: cfparams-autosilk-init
+create-source-security-groups: false
+create-ecr-repos: false
+restricted-accounts: false
+environments:
+  - continuous-integration
```

Comment:
```suggestion
environments: []
```
This might only need to be an empty list -- the CI resources are always deployed regardless of this list, and there isn't any other infrastructure that we need, right?

Reply:
i.e. Will we need a deploy role?

Reply:
Look at the `silk` pipeline that you mentioned modelling off, it looks like a no https://github.com/cultureamp/silk/blob/master/.buildkite/pipeline.yaml -- it appears it operates from the build role

Reply:
no sir. Just the one environment. If that's the case, I will drop this. Though I don't think "[]" is the accepted syntax for yaml 🤔

Reply:
if it uses the build role, that makes life easier then :D

Reply:
> Though I don't think "[]" is the accepted syntax for yaml

It is, but empty should work too. The only concern is the empty line is ambiguous -- it could be both `{}` or `[]` when parsed. Maybe the spec stipulates some order of precedence

Reply:
Welp, turns out it can't be empty anyway it seems:
![Uploading image.png…]()